app/assets/stylesheets/keywords.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Keywords controller here.
```

Comment:
How about removing this auto-generated file now as we are not using it also it is not aligning with our [CSS structure convention](https://nimblehq.co/compass/development/code-conventions/css/#stylesheets-structure).

Reply:
Thank you! Updated in [`067c022` (#37)](https://github.com/suho/google-search-ruby/pull/37/commits/067c0225bed0a50f9947e8747b64a6b210c6e73d)